pages/Blade-of-Vampires.md

```diff
@@ -0,0 +1,3 @@
+The Blade of Vampires is a [Weapon](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Weapons) added in Slimefun.<br>
+It is a Golden Sword which is enchanted with Fire Aspect II, Sharpness II, Unbreaking IV and Life Steal.<br>
```

Comment:
Either "golden sword" or "Gold Sword".

BTW you should probably mention that "Life Steal" isn't an actual enchantment, otherwise it may confuse people.